### PR TITLE
#164797980 user create article

### PR DESCRIPTION
**Pull request by AnayoOleru**
#### Description
Currently, the app does not have any page for users to create articles. This PR adds the page where users can create articles
#### Type of change

- [x] create components
- [x] add react and react-rdeux

#### How Has This Been Tested?

N/A

#### Checklist:
- [x] create components
- [x] add react and react-rdeux
#### PT-ID

[#164797980](https://www.pivotaltracker.com/n/projects/2318856)

#### Screenshots
Screen Shot 2019-05-09 at 2 49 07 PM
Screen Shot 2019-05-09 at 2 49 34 PM

#### Questions:
This PR is still under construction 

Due to the number of files change, I decided not to add test for this PR so it doesn't increase the file changes. But after this, I will write a test for it and push before working on the POST.

### CODE REVIEW

File: src/components/presentations/NewArticle/NewArticle.jsx [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+      headerpic: '',
+      editorState: EditorState.createEmpty(),
+    };
+    this.onChange = editorState => {
+      const contentState = editorState.getCurrentContent();
+      const html = stateToHTML(contentState);
+      const converted = (convertToRaw(contentState), html);
+      this.setState({ body: converted, editorState });
+    };
+    this.toggleBlockType = this.toggleBlockType.bind(this);
```
Comment (vic3king) [inline src/components/presentations/NewArticle/NewArticle.jsx]: can we remove the `bind(this)`? i think we can still achieve this by just doing `this.toggleBlockType` or we can use an arrow function `{() => this.toggleBlockType}`

Comment (rotimi-babalola) [reply src/components/presentations/NewArticle/NewArticle.jsx]: Actually using the `bind` syntax is better. Using an inline arrow function creates a new function on every render which is not good for performance. The other option could be to use an arrow function declared as a class property @vic3king

File: src/components/presentations/NewArticle/BlockStyles/BlockStyleToolbar.jsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+];
+
+export function getBlockStyle(block) {
+  switch (block.getType()) {
+    case 'blockquote':
+      return 'RichEditor-blockquote';
+    default:
+      return null;
+  }
+}
```
Comment (rotimi-babalola) [inline src/components/presentations/NewArticle/BlockStyles/BlockStyleToolbar.jsx]: Can you define all these in a separate file and import them here? @AnayoOleru

Comment (AnayoOleru) [reply src/components/presentations/NewArticle/BlockStyles/BlockStyleToolbar.jsx]: @rotimi-babalola I'll work on that now

**Pull request closed without merging**